tests/OptimizedPathMappingRepositoryTest.php

```diff
@@ -170,4 +105,33 @@ public function testRemoveFailsIfLanguageNotGlob()
     {
         $this->writeRepo->remove('/*', 'foobar');
     }
+
+    public function testAddRelativePathInStore()
+    {
+        $repo = $this->createBaseDirectoryRepository($this->store, __DIR__.'/Fixtures');
+        $repo->add('/webmozart/file', new FileResource(__DIR__.'/Fixtures/dir1/file1'));
+        $repo->add('/webmozart/dir', new DirectoryResource(__DIR__.'/Fixtures/dir2'));
+
+        $this->assertTrue($repo->contains('/webmozart/file'));
+        $this->assertTrue($repo->contains('/webmozart/dir'));
+        $this->assertEquals('dir1/file1', $this->store->get('/webmozart/file'));
+        $this->assertEquals('dir2', $this->store->get('/webmozart/dir'));
+        $this->assertPathsEquals(__DIR__.'/Fixtures/dir1/file1', $repo->get('/webmozart/file')->getFilesystemPath());
```

Comment:
`getFilesystemPath()` should already return a normalized path. Hence it should be sufficient to do:

```php
$this->assertSame(Path::normalize(__DIR__.'/Fixtures/dir1/file1'), $repo->get('/webmozart/file')->getFilesystemPath());
```

This is also the way I solved failing Windows tests in other repositories.

You can remove `assertPathsEquals()`.